bitnami/mongodb/templates/NOTES.txt

```diff
@@ -90,7 +90,7 @@ MongoDB(R) can be accessed on the following DNS name(s) and ports from within yo
 
 To get the root password run:
 
-    export MONGODB_ROOT_PASSWORD=$(kubectl get secret --namespace {{ template "mongodb.namespace" . }} {{ template "mongodb.fullname" . }} -o jsonpath="{.data.mongodb-root-password}" | base64 --decode)
+    export MONGODB_ROOT_PASSWORD=$(kubectl get secret --namespace {{ template "mongodb.namespace" . }} {{ .Values.auth.existingSecret || template "mongodb.fullname" . }} -o jsonpath="{.data.mongodb-root-password}" | base64 --decode)
```

Comment:
Hi @Startouf! Thanks for raising this improvement! This approach throws a parse error, however:
```
Error: parse error at (mongodb/templates/NOTES.txt:93): unexpected "|" in command
```
I would suggest an if/else approach like in
https://github.com/bitnami/charts/blob/07481c06d6f082595e8c9bc6151899ec10627f36/bitnami/mongodb/templates/NOTES.txt#L110-L114

Reply:
I found out a helper already existed (and it is the same across many charts) so I just used it in the notes instead